assets/README.md

```diff
@@ -0,0 +1,8 @@
+This folder contains various assets used in Oppia codebase.
```

Comment:
The folder structure is useful, but could you also talk a bit as well about how this file gets generated and what gets included/excluded? Devs should understand how this folder is generated and what the rules are for something being included/excluded in it.

Reply:
This folder is present as a permanent part of codebase, so we do not need to remove it. I am sorry but I am not sure what can I add about addition/removal of the folder in this case. Can you please provide an example for the same on what can I add here?

Reply:
Sorry, this is my bad. I was thinking of backend_prod_files when I wrote this comment -- please ignore.